File: python-00/ex2/ft_harvest_total.py
#!/usr/bin/env python3


def ft_harvest_total():
    total = 0
    day = int(input("Day 1 harvest: "))
    total += day
    day = int(input("Day 2 harvest: "))
    total += day
    day = int(input("Day 3 harvest: "))
    total += day
    print(f"Total harvest: {total}")


ft_harvest_total()

File: c-01/ex2/ft_harvest_total.c
#include <stdio.h>

void ft_harvest_total()
{
    int day = 0, total = 0;
    printf("Day 1 harvest: ");
    scanf("%d", &day);
    total += day;
    printf("Day 2 harvest: ");
    scanf("%d", &day);
    total += day;
    printf("Day 3 harvest: ");
    scanf("%d", &day);
    total += day;
    printf("Total harvest: %d\n", total);
}

int main(void)
{
    ft_harvest_total();
    return 0;
}
